dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/model/Contentlet.java

```diff
@@ -182,7 +183,17 @@ public Structure getStructure() {
     	structure = CacheLocator.getContentTypeCache().getStructureByInode(getStructureInode());
         return structure;
     }
+    
+    /**
+     */
+    public ContentType getContentType() {
```

Comment:
This was done to mirror existing functionality, but you are right.  Removed and deprecated old functionality.